docs/BR.md

```diff
@@ -1756,142 +1754,916 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
-
-#### 7.1.2.1 Root CA Certificate
-
-a. `basicConstraints`
-
-   This extension MUST appear as a critical extension. The `cA` field MUST be set true. The `pathLenConstraint` field SHOULD NOT be present.
-
-b. `keyUsage`
-
-   This extension MUST be present and MUST be marked critical. Bit positions for `keyCertSign` and `cRLSign` MUST be set. If the Root CA Private Key is used for signing OCSP responses, then the `digitalSignature` bit MUST be set.
-
-c. `certificatePolicies`
-
-   This extension SHOULD NOT be present.
-
-d. `extKeyUsage`
-
-   This extension MUST NOT be present.
-
-#### 7.1.2.2 Subordinate CA Certificate
-
-a. `certificatePolicies`
-
-   This extension MUST be present and SHOULD NOT be marked critical.
-
-   `certificatePolicies:policyIdentifier` (Required)
-
-   The following fields MAY be present if the Subordinate CA is not an Affiliate of the entity that controls the Root CA.
-
-   * `certificatePolicies:policyQualifiers:policyQualifierId` (Optional)
-
-     `id-qt 1` [RFC5280].
-
-   * `certificatePolicies:policyQualifiers:qualifier:cPSuri` (Optional)
-
-     HTTP URL for the Root CA's Certificate Policies, Certification Practice Statement, Relying Party Agreement, or other pointer to online policy information provided by the CA.
-
-b. `cRLDistributionPoints`
-
-   This extension MUST be present and MUST NOT be marked critical. It MUST contain the HTTP URL of the CA's CRL service.
+### 7.1.2 Certificate Content and Extensions
+
+If the CA asserts compliance with these Baseline Requirements, all certificates that it issues MUST comply with one of the following certificate profiles:
+
+  * CA Certificates
+    * [Section 7.1.2.1.- Root CA Certificate Profile](#7121-root-ca-certificate-profile)
+    * Subordinate CA Certificates
+      * Cross Certificates
+        * [Section 7.1.2.2 - Cross-Certified Subordinate CA Certificate Profile](#7122-cross-certified-subordinate-ca-certificate-profile)
+      * Technically Constrained CA Certificates
+        * [Section 7.1.2.3 - Technically-Constrained Non-TLS Subordinate CA Certificate Profile](#7123-technically-constrained-non-tls-subordinate-ca-certificate-profile)
+        * [Section 7.1.2.4 - Technically-Constrained TLS Subordinate CA Certificate Profile](#7124-technically-constrained-tls-subordinate-ca-certificate-profile)
+      * [Section 7.1.2.5 - TLS Subordinate CA Certificate Profile](#7125-tls-subordinate-ca-certificate-profile)
+  * [Section 7.1.2.6 - Subscriber (End-Entity) Certificate Profile](#7126-subscriber-server-certificate-profile)
+    * Domain Validated
+    * Individual Validated
+    * Organization Validated
+    * Extended Validation
+  * __**TBD: Infrastructure Certificates**__
+    * [Section 7.1.2.7 - OCSP Responder Certificate Profile](#7127-ocsp-responder-certificate-profile)
+    * __**TBD: Infrastructure Cert**__
+    * __**TBD: Precert Signing CA**__
```

Comment:
Several CAs currently use Precertificate Signing Certificates, so either the profile needs to be fleshed out separately or one of the above profiles should mention it should be used for Precert Signers.